picard/tagger.py

```diff
@@ -1246,12 +1246,52 @@ def sighandler(self):
         self.signalnotifier.setEnabled(True)
 
 
-def version():
-    print("%s %s %s" % (PICARD_ORG_NAME, PICARD_APP_NAME, PICARD_FANCY_VERSION_STR))
```

Comment:
Why is this disappeared?
Those methods are required.

Reply:
Where is it used? I cannot find any usage

Reply:
So basically that's why I moved the logic to `PicardArgs` :)

Reply:
I believe that move is also the reason for the failing tests.

Reply:
> Where is it used? I cannot find any usage

On master:

```bash
❯ python tagger.py -V             
Picard 2.9.0.dev1, Python 3.10.4, PyQt 5.15.7, Qt 5.15.2, Mutagen 1.45.1, Discid discid 1.2.0, libdiscid 0.6.2, astrcmp Python, SSL OpenSSL 1.1.1m  14 Dec 2021
❯ python tagger.py -v
MusicBrainz Picard 2.9.0.dev1

```

On this branch, behavior changed and Picard starts instead.

Reply:
The problem is that you changed lines 1353-1355.

Reply:
Okay, I see that I forgot to assign the value to the `PicardArgs.version` and it always remained the default `""` empty string

Reply:
What output do you get when you start it with the `--long-version ` command line option?

Reply:
@rdswift It works exactly the same for the end user, but the functions are no longer stand-alone code but wrapped in the class.

Reply:
See PicardArgs. __get_version(), PicardArgs.version value depends on the flag provided by the user.

Reply:
Okay, thanks.

Reply:
Thinking about this a bit more...  Not that it makes a big difference, but I sort of question the need to store the version string in a variable when it is only used once.  Then again, maybe it is stored as an empty string if neither `-v` nor `-V` are specified on the command line, and Picard exits after displaying the string when `-v` or `-V` are specified.

Reply:
> I sort of question the need to store the version string in a variable when it is only used once. Then again, maybe it is stored as an empty string if neither -v nor -V are specified on the command line, and Picard exits after displaying the string when -v or -V are specified.

If we wanted to introduce just `PicardArgs.get_version()`, we'd end up having to store the booleans from `argparse` to determine which wersion actually return, so we still end up with 2 variables. And since @zas advised me not to overuse `del`, I left it there.